plugins/plugin-chart-table/src/DataTable/DataTable.tsx

```diff
@@ -193,6 +199,8 @@ export default function DataTable<D extends object>({
     return (wrapStickyTable ? wrapStickyTable(getNoResults) : getNoResults()) as JSX.Element;
   }
 
+  const totalsHeaderSpan = totals && columnsMeta.length - Object.keys(totals).length;
```

Comment:
Here we need to remove percentage metrics, otherwise regular metrics won't show up:
```suggestion
  const totalsHeaderSpan =
    totals && columnsMeta.filter(col => !col.isPercentMetric).length - Object.keys(totals).length;
```
before:
![image](https://user-images.githubusercontent.com/33317356/112979440-41feb980-9161-11eb-99ee-7311a270eeef.png)

after:
![image](https://user-images.githubusercontent.com/33317356/112979357-23002780-9161-11eb-9545-2076fc94f20f.png)